Cartfile

```diff
@@ -14,8 +14,10 @@ github "mozilla-mobile/MappaMundi"          "master"
 github "Leanplum/Leanplum-iOS-SDK"          ~> 2.7.2
 github "mozilla-services/shavar-prod-lists" "72.0"
 # Use release version of application-services
-github "mozilla/application-services"      "v61.0.12"
```

Comment:
Quick note: Since these changes are only meant for XCode 12 it would be nice to have them merge in a different PR and not have it in this one. Mainly thinking about our main branch that is still on XCode 11.